messages/6_jsonld_vp_token.json

```diff
@@ -48,15 +48,15 @@
                 }
             }
         ],
-        "definition_id": "NextcloudCombinedRequest",
+        "definition_id": "d76c51b7-ea90-49bb-8368-6b3d194fc130",
         "id": "NexcloudCredentialPresentationSubmission"
     },
     "proof": {
         "type": "BbsBlsSignature2020",
         "verificationMethod": "did:key:zUC7...",
         "created": "2022-05-04T14:02:36.045015+00:00",
         "proofPurpose": "authentication",
-        "challenge": "c17e7680-2030-42ae-95d4-ef9ef320f8f6",
+        "challenge": "22456608037449564223",
```

Comment:
soweit ich verstanden habe, funktioniert das nicht mit aca-by?
"bei der Implementation des Nextcloud Plugins ist mir aufgefallen, dass das Indy SDK nur Integer Noncen im Stringformat akzeptiert (z.b. "32281928655741385930") und der aca-py nur UUID4 Noncen verarbeiten kann"

Reply:
Ja das stimmt. Allerdings ist hier die entgültige Lösung ja noch nicht klar, da wir auch den aca-py entsprechend anpassen könnten. Ich habe hier nur die Werte getauscht, weil die Nonce aus dem Authentication Request in der Presentation und nicht dem Credential verarbeitet wird. Das war zuvor falsch.

Reply:
ok. Prinzipiell richt, würde aber so aktuell nicht funktionieren.